src/sidebar/components/timestamp.js

```diff
@@ -15,14 +15,21 @@ function TimestampController($scope, time) {
   var self = this;
 
   function updateTimestamp() {
-    self.relativeTimestamp = time.toFuzzyString(self.timestamp);
-    self.absoluteTimestamp = dateUtil.format(new Date(self.timestamp));
+    //  This is (should be) a temporary solution.
+    var modifiedTime = self.timestamp.replace(/ /g,'T').concat('Z');
+    /* The 'updated' and 'created' data are sent to server in a format of (YYYY-MM-DDTMM:SS:SS.SSZ)
+    * However, somehow it is returned in a format of (YYYY-MM-DD MM:SS:SS.SS) (No T and Z) and this causes
```

Comment:
If this is a bug on the server side, don't fix this - we'll fix it server side.

Reply:
I think it is. I'm going to revert this changes and create a ticket for you.

Reply:
I completed this work in the CPDRUPAL ticket. there's n MR out for you to test the branch and validate I haven't broken anything.